packages/project-management-automation/lib/add-milestone.js

```diff
@@ -3,6 +3,12 @@
  */
 const debug = require( './debug' );
 
+/** @typedef {import('@octokit/rest').HookError} HookError */
+/** @typedef {import('@actions/github').GitHub} GitHub */
+/** @typedef {import('@octokit/webhooks').WebhookPayloadPush} WebhookPayloadPush */
+/** @typedef {import('@octokit/rest').Response} OctokitResponse */
```

Comment:
I don't see `OctokitIssuesListMilestonesForRepoResponse ` and `OctokitResponse` referenced in this file and the package in general. Are they necessary to include?

Reply:
> I don't see `OctokitIssuesListMilestonesForRepoResponse ` and `OctokitResponse` referenced in this file and the package in general. Are they necessary to include?

It's not. It was a bad copy-paste on my part 😅 

Fixed in b6c1af870.